modules/bpi/bpi.api.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+/**
+ * @file
+ * Describe hooks provided by the BPI module.
+ */
+
+/**
+ * @addtogroup hooks
+ * @{
+ */
+
+/**
+ * Return a list of BPI node types a module can handle.
+ *
+ * @return array
```

Comment:
So this is basicly `string[]`.

Reply:
Yes. I'll update the doc to reflect this.